.circleci/config.yml

```diff
@@ -79,6 +79,9 @@ jobs:
       - run:
           name: Setup manifest
           command: python3 setup.py ${BROWSER}
+      - run:
```

Comment:
Following the rebase on #56 ; this go moved into the new build-ext job, but should be part of the basic-testing job.
Please move it following this.